firstvoices/backend/views/search/base_search_views.py

```diff
@@ -29,7 +29,7 @@
             200: inline_serializer(
                 name="SearchResults",
                 fields={
-                    "id": serializers.CharField(),
+                    "searchResultId": serializers.CharField(),
```

Comment:
Just for clarification, will this field display an id other than the searched model's id?

Reply:
This fields will display the `id` that ES uses internally, while the `id` field in the `entry` object represents the uuid from the postgres db. This field can help as a key when listing objects for react to avoid the duplicate key warning.